panda/src/navigation/navMesh.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * PANDA 3D SOFTWARE
+ * Copyright (c) Carnegie Mellon University.  All rights reserved.
+ *
+ * All use of this software is subject to the terms of the revised BSD
+ * license.  You should have received a copy of this license along
+ * with this source code in a file named "LICENSE."
+ *
+ * @file navMesh.h
+ * @author ashwini
+ * @date 2020-060-21
+ */
+
+
+#ifndef NAVMESH_H
+#define NAVMESH_H
+
+#include "Recast.h"
+#include "DetourNavMesh.h"
+#include "typedWritableReferenceCount.h"
+
+class EXPCL_NAVIGATION NavMesh: public TypedWritableReferenceCount
```

Comment:
Class lacks docstring.

Reply:
Should docstrings be added in header files as well? I have added them in cxx files only for all the functions @rdb